lib/chef/client.rb

```diff
@@ -55,7 +55,6 @@
 require "chef-utils" unless defined?(ChefUtils::CANARY)
 require "ohai" unless defined?(Ohai::System)
 require "rbconfig" unless defined?(RbConfig)
-require_relative "dist"
```

Comment:
Looks like maybe I should have this changed to the following?
```ruby
require "chef-utils/dist"
```

Reply:
yeah you should need to change that, that file shouldn't exist so something should be blowing up in tests?

Reply:
Actually this is covered with via the require on ``chef-utils`` above so this can be ignored.